modules/cert-manager-install-console.adoc

```diff
@@ -24,12 +24,12 @@ You can use the web console to install the {cert-manager-operator}.
 . Select the *{cert-manager-operator}* and click *Install*.
 
 . On the *Install Operator* page:
```

Comment:
```suggestion
. On the Install Operator page:
```

I don't think we need to highlight page names. In this particular case, the page name uses headline-style capitalization, so that should suffice.

Reply:
Page names are always bolded

Reply:
Thanks for letting me know! That isn't the case for IBM Style, and I checked the SSG, but didn't see anything there about it. I'll remember that going forward.